### DISCUSSION TITLE
Keyboard highlight random colors

### DISCUSSION THREAD

**Discussion by Mikey-Cai** [seerge/g-helper#2724] [Ideas]
in the armoury create, keyboard highlight can choose random colours, but in g-helper i can only choose one color，maybe can u update, thank u

**Comment by seerge** [comment]
@Mikey-Cai hello, you can't really do that as app mainly shows only backlight modes supported by your keyboard on hardware level. You can use rainbow or color-cycle tho, they have a similar effect, if I understand you correctly? :)

**Comment by Mikey-Cai** [comment]
Ha ha ha. At home I use rainbow or color-cycle tho, but at the office I still want to use HIGHLIGHT or I'll get attention from my boss hahaha.Actually I mean this random color. In g-helper , maybe can you add random colour:)

Thanks again










At 2024-06-13 15:35:09, "Serge" ***@***.***> wrote:

@Mikey-Cai hello, you can't really do that as app mainly shows only backlight modes supported by your keyboard on hardware level. You can use rainbow or color-cycle tho, they have a similar effect, if I understand you correctly? :)
